### PR TITLE
Use stateId instead of location/_location

### PR DESCRIPTION
**Pull request by igorkofman**

### CODE REVIEW

File: src/components/ShareBlock/ShareModelBlock.js [hunk 1]
```diff
@@ -4,13 +4,11 @@ import { STATES } from 'enums';
 
 import EmbedPreview from './EmbedPreview';
 
-const ShareModelBlock = ({ condensed, location, county }) => {
-  const locationName = STATES[location];
+const ShareModelBlock = ({ condensed, stateId, county }) => {
```
Comment (mikelehen) [inline src/components/ShareBlock/ShareModelBlock.js:7]: You changed this, but not the code that references it in Embed.js.

So if you go to http://localhost:3000/us/wa and click the "EMBED" share button, and then click the share button inside the embed iframe, you get an error.

More stuff TypeScript could catch. :-)

File: src/screens/ModelPage/ModelPage.js [hunk 2]
```diff
@@ -62,37 +64,34 @@ function ModelPage() {
...
-    actionTitle = `${locationName}: Urge your public officials to act now against the COVID-19 outbreak!`;
+    actionTitle = `${stateName}: Urge your public officials to act now against the COVID-19 outbreak!`;
     actionDescription = `To prevent hospital overload, our projections indicate a Stay at home order must be implemented soon.`;
   }
   let metaTags = (
     <AppMetaTags
-      canonicalUrl={`/us/${_location.toLowerCase()}`}
-      pageTitle={`${locationName} Forecast`}
+      canonicalUrl={`/us/${stateId.toLowerCase()}`}
+      pageTitle={`${stateName} Forecast`}
```
Comment (mikelehen) [inline src/screens/ModelPage/ModelPage.js:94]: Huh.  I guess none of these meta tags respect the county at all.  Logged a bug. https://github.com/covid-projections/covid-projections/issues/568

File: src/components/ShareBlock/ShareModelBlock.js (hunk 1 above)
Comment (igorkofman) [reply src/components/ShareBlock/ShareModelBlock.js:7]: Share buttons inside share button  🤦  Thanks for the careful review!

File: src/components/Charts/ModelChart.style.js [hunk 3]
```diff
@@ -224,6 +224,19 @@ export const Wrapper = styled.div`
...
+      )} {
+      background: ${props =>
+        props.projections.getAlarmLevelColor() === COLOR_MAP.GREEN.BASE
+          ? COLOR_MAP.GREEN.DARK
+          : props.projections.getAlarmLevelColor()};
+    }
+
+    &.custom-plot-label-${snakeCase(INTERVENTIONS.SHELTER_IN_PLACE)} {
+      background: ${props => props.projections.getAlarmLevelColor()};
+    }
```
Comment (mikelehen) [inline src/components/Charts/ModelChart.style.js:239]: I'm guessing this is the iffy css merge you mentioned?

Comment (igorkofman) [reply src/components/Charts/ModelChart.style.js]: Yep, fixed.

**Pull request merged**